.github/workflows/stale.yml

```diff
@@ -2,7 +2,7 @@ name: stale
 
 on:
   schedule:
-    - cron: '0 15 * * *'
+    - cron: '0 23 * * 7,1,2,3,4'
```

Comment:
FYI
It also works.

```
'0 23 * * SUN,MON,TUE,WED,THU'
```

```
'0 23 * * SUN-THU'
```

Reply:
I applied your second suggestion. Thank you!